hardhat/hardhat.config.ts

```diff
@@ -27,16 +27,16 @@ const config: HardhatUserConfig = {
     geth: {
       url: 'http://localhost:8545',
       chainId: 1337,
-      accounts: PRIVATE_KEYS,
+      accounts: [getAccount('admin').privateKey]
```

Comment:
I have my suspicions that these may affect the development setup of the blockchain node.

Could you clear, run, fund and boot to verify this did not impact anything there?

Reply:
Just checked and run, fund, and boot for the chain all work with this change in local development.

The reason I think this is safe is the deploy scripts (https://github.com/Lilypad-Tech/lilypad/tree/main/hardhat/deploy) only deploy and excute commands from `admin`.

The public addresses for the other users appear to be configured here:

https://github.com/Lilypad-Tech/lilypad/blob/afbf4f8f9e45acca16615ffe730f19e2dfad2553/hardhat/hardhat.config.ts#L24

And they are referenced, but admin does all the work.

Reply:
Thank you for verifying.